lochness/utils/source_check.py

```diff
@@ -311,21 +311,28 @@ def check_source(Lochness: 'lochness', test: bool = False) -> None:
 
     # select final_check failed files, and clean up
     qc_fail_df = all_df[
-            (~all_df['final_check']) | (~all_df['exist_in_db'])
+            (~all_df['final_check']) | 
+            (~all_df['exist_in_db']) |
+            (~all_df['subject_check'])
             # (~all_df['file_name'].str.contains('.dcm|._dicom_series'))
             ]
 
+    qc_fail_df['subject_check'] = qc_fail_df['subject_check'].map(
+            {True: f'Correct', False: f'Incorrect'})
+
     qc_fail_df['exist_in_db'] = qc_fail_df['exist_in_db'].map(
             {True: f'Exist in {db_string}', False: f'Missing in {db_string}'})
 
     qc_fail_df['final_check'] = qc_fail_df['final_check'].map(
             {True: 'Correct', False: 'Incorrect'})
 
     cols_to_show = ['file_path', 'site', 'subject', 'modality',
-                    'exist_in_db', 'final_check']
+                    'subject_check', 'exist_in_db', 'final_check']
+
     qc_fail_df = qc_fail_df[cols_to_show]
     qc_fail_df.columns = ['File Path', 'Site', 'Subject',
-                          'Data Type', 'Subject ID in database', 'Format']
+                          'Data Type', 'AMPSCZ-ID checksum',
+                          'Subject ID in database', 'Format']
```

Comment:
What is an example of `AMPSCZ-ID checksum`?

Reply:
`Correct` or `Incorrect`